teamengine-resources/src/main/resources/com/occamlab/te/earl_html_report.xsl

```diff
@@ -205,6 +196,28 @@
                 <xsl:value-of select="count(//earl:TestRequirement/cite:testsFailed[text() &gt;'0'])" />
               </div>
               <!-- Pass:  <xsl:value-of select="rdf:RDF/cite:TestRun/cite:testsPassed"/> | Fail: <xsl:value-of select="rdf:RDF/cite:TestRun/cite:testsFailed"/> | Skip: <xsl:value-of select="rdf:RDF/cite:TestRun/cite:testsSkipped"/> -->
+              Core Conformance Classes (Pass = Green; Fail = Red; Skip = Grey):
+              <xsl:for-each select="/rdf:RDF/cite:TestRun/cite:requirements/rdf:Seq/rdf:li/earl:TestRequirement">
+                <xsl:if test="cite:isBasic = 'true'">
+                <xsl:variable name="testsInheritedFailure" select="if (cite:testsInheritedFailure >= 0) then cite:testsInheritedFailure else 0 " />
+                <xsl:variable name="bgcolor" >
+					<xsl:choose>
+						<xsl:when test="cite:testsPassed > 0 and cite:testsFailed = 0 and $testsInheritedFailure = 0">
+							 <xsl:value-of select="string('#B2F0D1')" />
+						</xsl:when>
+						<xsl:when test="cite:testsPassed = 0 and cite:testsFailed > 0 or $testsInheritedFailure > 0">

```

Comment:
"cite:testsPassed = 0" seems to be not correct as a conformance class is failed even if single tests passed.